### DISCUSSION TITLE
AM modulation

### DISCUSSION THREAD

**Discussion by pschatzmann** [jgaeddert/liquid-dsp#288] [How do I....?]
I found [bitlunis ESP32 AM Transmitter](https://bitluni.net/am-radio-transmitter) and I was wondering if I can implement this with your library on an ESP32 using Arduino. 
I am a little bit worried that your library is using floating point arithmetic but nevertheless I want to give it a try. 

The processing chain looks as follows:
- input is an audio signal usually using a sample rate of 44100.
- This signal should be AM modulated with a carrier frequency of 835kHz and a sample rate of ~13MHz.

I looked at your ampmodem_example.c but somehow I can't relate the parameters with my case.

Any help is appreciated...

**Comment by jgaeddert** [comment]
Fascinating! Are you trying to generate samples real-time? or just pre-generated like in the video?

**Comment by pschatzmann** [comment]
I will try to use my [Arduino Audio Tools](https://github.com/pschatzmann/arduino-audio-tools) library that supports many audio input sources. I plan to test it with some mp3 files stored on an SD card.
